apps/ca.c

```diff
@@ -821,7 +821,7 @@ int ca_main(int argc, char **argv)
         }
         if (verbose)
             BIO_printf(bio_err, "message digest is %s\n",
-                       OBJ_nid2ln(EVP_MD_type(dgst)));
+                       EVP_MD_description(dgst));
```

Comment:
This can be (and currently will be) NULL, won't it?

Reply:
Would `EVP_MD_name()` be a better choice here?

Reply:
Yep, I think so.

Reply:
And here as well. So you can drop these hunks.